### DISCUSSION TITLE
How to reset the InteractiveViewer content scale

### DISCUSSION THREAD

**Discussion by marc-svg** [flet-dev/flet#4387] [Q&A] [answered]
Hi,
When using an InteractiveViewer to resize, pan an Image how do I reset the image to its initial size and position ?

What is the InteractiveViewer's attribute that resets the content size (when resized) and position (when paned) ?
What is the InteractiveViewer's attribute that is updated when I resize /pan the content ?

Thanks

**Comment by ndonkoHenri** [comment, marked as answer]
I created an issue to make this possible: https://github.com/flet-dev/flet/issues/4391

**Comment by marc-svg** [comment]
Thank you


Le 18 nov. 2024 à 13:00, TheEthicalBoy ***@***.***> a écrit :

﻿

I created an issue to make this possible: #4391<https://github.com/flet-dev/flet/issues/4391>
